### ISSUE TITLE
get_form() not adding action to the object

### ISSUE THREAD

**Issue by cdvv7788** [mirumee/django-payments#97]
I am following the documentation and getting the form on my view with 
```python
payment.get_form(request.POST or None)
```
That object should have an attribute named action, for use in the context:

```html
<form action="{{ form.action }}" method="{{ form.method }}">
    {{ form.as_p }}
    <p><input type="submit" value="Proceed" /></p>
</form>
```

Action is None for the backends i tried(Dummy, authorize.net, stripe). It should return something valid if possible, and None is not. A blank string or request.path would make more sense maybe?

**Comment by patrys** [comment]
Huh, the base `PaymentForm` has exactly that: https://github.com/mirumee/django-payments/blob/master/payments/forms.py#L23

Something is not right. What type is the object returned?

**Comment by cdvv7788** [comment]
action is None. I don't know if it is related to validation or something.
payment.get_form() for stripe provider returns payments.stripe.forms.ModalPaymentForm, however no action is defined.
It seems https://github.com/mirumee/django-payments/blob/master/payments/forms.py#L36 is overriding that somehow. The action argument is getting lost somewhere.

**Comment by cdvv7788** [comment]
I would say change action=None to action='', as that is intended to be the default setting. What are the backends that should return an action other than blank?

**Comment by patrys** [comment]
You are right, it sets `action` to `None` by default which never makes sense.

**Comment by patrys** [comment]
Should be fixed with 0.8.2

**Comment by cdvv7788** [comment]
Nice. Thanks.
